src/ports/postgres/modules/deep_learning/madlib_keras.py_in

```diff
@@ -319,6 +308,38 @@ def fit(schema_madlib, source_table, model,model_arch_table,
     #TODO add a unit test for this in a future PR
     reset_cuda_env(original_cuda_env)
 
+def get_source_summary_table_dict(fit_validator):
+    source_summary = plpy.execute("""
+            SELECT
+                {class_values} AS class_values,
+                {norm_const} AS norm_const,
+                {dep_vartype} AS dep_vartype,
+                {dep_varname} AS dependent_varname_in_source_table,
+                {indep_varname} AS independent_varname_in_source_table
+            FROM {tbl}
+        """.format(class_values=CLASS_VALUES_COLNAME,
+                   norm_const=NORMALIZING_CONST_COLNAME,
+                   dep_vartype=DEPENDENT_VARTYPE_COLNAME,
+                   dep_varname='dependent_varname',
```

Comment:
don't we have variables for indep and dep varname constants ?

Reply:
I think the `dependent_varname` variable is equal to `MINIBATCH_OUTPUT_DEPENDENT_COLNAME_DL` (the column name of the dependent variable in the minibatched table), whereas the string `'dependent_varname'` here refers to the column name in the minibatched summary table that stores the value of the dependent variable column name (say `y`) in the original non-minibatched table.
Confusing, but this is another refactor JIRA by itself I guess!